tests/native/test_aarch64cpu.py

```diff
@@ -16,14 +16,28 @@
 from manticore.utils.fallback_emulator import UnicornEmulator
 from tests.native.test_aarch64rf import MAGIC_64, MAGIC_32
 
-ks = Ks(KS_ARCH_ARM64, KS_MODE_LITTLE_ENDIAN)
+from tests.native.aarch64cpu_asm_cache import assembly_cache
 
+ks = None
+
+
+def _ks_assemble(asm):
+    global ks
+    from keystone import Ks, KS_ARCH_ARM64, KS_MODE_LITTLE_ENDIAN
+
+    if ks is None:
+        ks = Ks(KS_ARCH_ARM64, KS_MODE_LITTLE_ENDIAN)
```

Comment:
```suggestion
def _ks_assemble(asm: str) -> bytes:
    """Assemble the given string using Keystone."""
    # Explicitly uses late importing so that Keystone will only be imported if this is called.
    # This lets us avoid requiring installation of Keystone for running tests.
    global ks
    from keystone import Ks, KS_ARCH_ARM64, KS_MODE_LITTLE_ENDIAN

    if ks is None:
        ks = Ks(KS_ARCH_ARM64, KS_MODE_LITTLE_ENDIAN)
```